CRM/MembershipExtras/Service/MembershipInstalmentsSchedule.php

```diff
@@ -121,10 +125,18 @@ public function generate(DateTime $startDate = NULL, DateTime $endDate = NULL, D
       }
     }
 
+    $instalments['sub_total'] = $this->getInstalmentsSubTotalAmount($instalments['instalments']);
+    $instalments['tax_amount'] = $this->getInstalmentsTotalTaxAmount($instalments['instalments']);
```

Comment:
Maybe using `getInstalmentsTaxAmount` instead of  `getInstalmentsTotalTaxAmount` is a little better.

Reply:
Good spot!